Dockerfile

```diff
@@ -1,33 +1,37 @@
-FROM python:3.7
+FROM python:3.9.5

```

Comment:
I think we agreed on 3.9.6 for a different project and should probably use that here as well.